### PR TITLE
Added gems

### PR DESCRIPTION
**Pull request by mikedao**
Added gems for production.
- Rspec-Rails
- better_errors
- capybara
- launchy 
- simplecov
- figaro
- awesome_print
- quiet_assets

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,45 +1,30 @@
...
-# Use jquery as the JavaScript library
-gem 'jquery-rails'
-# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
-gem 'turbolinks'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.0'
-# bundle exec rake doc:rails generates the API under doc/api.
-gem 'sdoc', '~> 0.4.0', group: :doc
+gem "jquery-rails"
+gem "turbolinks"
```
Comment (EmilyMB) [inline Gemfile:12]: Maybe delete this?

File: Gemfile [hunk 2]
```diff
@@ -1,45 +1,30 @@
...
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
-
-  # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
-
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem "byebug"
```
Comment (EmilyMB) [inline Gemfile:21]: I usually use pry instead of byebug.

File: Gemfile (hunk 1 above)
Comment (mikedao) [inline Gemfile:12]: Just Turbolinks? 

Will it hurt anything?

Comment (EmilyMB) [inline Gemfile:12]: Just search in the project for turbo and delete all occurrences.

Comment (mikedao) [inline Gemfile:12]: But _why_? DHH loves Turbolinks.

File: Gemfile (hunk 2 above)
Comment (mikedao) [inline Gemfile:21]: We can add pry, anyone have any objections to this?

Comment (nathanows) [inline Gemfile:21]: image

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+require "simplecov"
+SimpleCov.start
+require "rails_helper"
+require "capybara/rspec"
+
+RSpec.configure do |config|
+
```
Comment (mikedao) [inline spec/spec_helper.rb]: This has been corrected. Ignore, Hound complaint.

File: Gemfile (hunk 2 above)
Comment (mikedao) [inline Gemfile:21]: I added Pry.

File: Gemfile (hunk 1 above)
Comment (scottcrawford03) [inline Gemfile:12]: what was decided here?

File: Gemfile (hunk 2 above)
Comment (scottcrawford03) [inline Gemfile:21]: good, glad you did.

**Comment by scottcrawford03** [comment]
looks good to me. let's get it started :watch:

**Pull request merged**